Tests/SwiftSyntaxTest/VisitorTest.swift

```diff
@@ -49,12 +49,30 @@ public class SyntaxVisitorTestCase: XCTestCase {
       }
     }
     XCTAssertNoThrow(try {
-      let parsed = try SourceFileSyntax.parse(getInput("near-empty.swift"))
+      let parsed = try SyntaxTreeParser.parse(getInput("near-empty.swift"))
       let rewriter = VisitAnyRewriter(transform: { _ in
          return SyntaxFactory.makeIdentifier("")
       })
       let rewritten = rewriter.visit(parsed)
       XCTAssertEqual(rewritten.description, "")
     }())
   }
+
+  public func testSyntaxRewriterVisitCollection() {
+    class VisitCollections: SyntaxVisitor {
+      var numberOfCodeBlockItems = 0
+
+      override func visit(_ items: CodeBlockItemListSyntax) {
+        numberOfCodeBlockItems += items.count
+        super.visit(items)
+      }
+    }
+
+    XCTAssertNoThrow(try {
+      let parsed = try SyntaxTreeParser.parse(getInput("nested-blocks.swift"))
+      let visitor = VisitCollections()
+      visitor.visit(parsed)
+      XCTAssertEqual(4, visitor.numberOfCodeBlockItems)
```

Comment:
You and @harlanhaskins have different points of view about expected/actual argument order 🙂  (I don't have a preference either way, so I'll change it since I've already changed the others.)

Reply:
Oh, I'm with @ahoppen on this. The only reason I had them as expected-first is because that's the order StdlibUnittest expected them, which made me sad.